app/WebParsing/UtsgJsonParser.hs

```diff
@@ -11,10 +11,8 @@ import Data.Aeson.KeyMap as KM hiding (insert, map)
 import qualified Data.HashMap.Strict as HM
 import Data.Maybe (catMaybes)
 import qualified Data.Text as T
-import Database.Persist.Sqlite (Filter, SqlPersistM, deleteWhere, insert, insertMany_, runSqlite,
-                                selectFirst, (==.))
-import Database.Tables (Courses (..), EntityField (CoursesCode), MeetTime (..), Meeting (..),
-                        Times (..), buildTimes)
+import Database.Persist.Sqlite
```

Comment:
Don't remove named imports. It's always better to be explicit with imports (moreover, in general you should not be changing code style like this).